packages/template-retail-react-app/package.json

```diff
@@ -78,7 +78,7 @@
         "start:inspect": "npm run start -- --inspect",
         "start:pseudolocale": "npm run extract-default-translations && npm run compile-translations:pseudo && cross-env USE_PSEUDOLOCALE=true npm run start",
         "tail-logs": "pwa-kit-dev tail-logs",
-        "test": "pwa-kit-dev test",
+        "test": "pwa-kit-dev test --",
```

Comment:
what is this change for?

Reply:
Sorry I did not mean to commit that. Reverted!